GUI/src/main/kotlin/Timeline.kt

```diff
@@ -54,6 +54,7 @@ class Timeline {
                 )
             }
 //            #### timeline box ####
+            var helper = Offset.Zero
```

Comment:
Could you rename this variable to a more descriptive one? I think, it is used to store the actual mouse position, so maybe `mousePosition`?